.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    "superjson-next" // 👈
+  ]
```

Comment:
This gets around an issue with NextJS `getStaticProps` serializable requirements, and Prisma returns some `Date` objects that aren't. See https://github.com/blitz-js/superjson#using-with-nextjs